source/features/hide-diff-signs.css

```diff
@@ -0,0 +1,4 @@
+.rgh-hide-diff-signs .blob-code-inner::before, /* Inline review blocks in PRs */
+.rgh-hide-diff-signs .blob-code-marker-cell::before { /* Commit page */
```

Comment:
Not related to this PR but I think this feture is broken

Reply:
Is it broken or is it #4695

Reply:
:shrug: I am not by a computer to check.

Reply:
It is!

Reply:
![smdu0s5y16i71](https://user-images.githubusercontent.com/1402241/130363363-fdf1dcf3-ca3e-4bcf-801d-0d3d689f9895.jpg)